swift/ios-shoppe-demo/ModelControllers/OrderManger.swift

```diff
@@ -0,0 +1,48 @@
+//
+//  OrderManger.swift
+//  ios-shoppe-demo
+//
+//  Created on 4/1/20.
+//  Copyright © 2020 FullSTory All rights reserved.
+//
+
+import Foundation
+
+class OrderManager {
+    static var sharedInstance = OrderManager()
+
+    var currentOrder: Order?
+    var itemQuantities = [String: Int]()
```

Comment:
Having `itemQuantities` hold information here means that we have data in the OrderManager itself and also in `currentOrder`.  Given the `Order` object is pretty shallow, does it make more sense for it to also hold item quantity data?  The approach I took in the Angular app was to forego an order model and simply create a list of products in the [manager](https://github.com/fullstorydev/angular-shoppe-demo/blob/master/src/app/services/cart.service.ts).  The `quantity` property on the `Product` doubles as inventory and item quantities.

Reply:
🧐